server/routers/deliverables.js

```diff
@@ -1,16 +1,22 @@
 const express = require('express')
 const router = express.Router();
+
+const context = require('../entities/database/context')
+const Deliverable = context.Deliverable
+const Project = context.Project
+
 const authMiddleware = require('../middleware/authentication-middleware');
 const apiError = require('../entities/api-error');
 
 // TODO: uncomment and test after session router is made
```

Comment:
```suggestion
```

Reply:
Cannot easily test that right now as there is no route to register users and the password field in the database should be a hash